### DISCUSSION TITLE
Limit Concurrent Execution

### DISCUSSION THREAD

**Discussion by rawlinns-dev** [quartznet/quartznet#1368] [Q&A] [answered]
Is it possible to limit the number of concurrent executions of a given job ?

[DisallowConcurrentExecution] limits this to one

Maybe something similar to https://github.com/Evolveum/quartz/tree/quartz-2.3.0.e2 Execution Groups.

**Comment by lahma** [comment, marked as answer]
Yes there is an issue about the feature https://github.com/quartznet/quartznet/issues/1175
